docs/docs/jsx-in-depth.md

```diff
@@ -228,6 +236,18 @@ If you pass no value for a prop, it defaults to `true`. These two JSX expression
 
 In general, we don't recommend using this because it can be confused with the [ES6 object shorthand](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015) `{foo}` which is short for `{foo: foo}` rather than `{foo: true}`. This behavior is just there so that it matches the behavior of HTML.
 
+### Props with 'False' values
+
+If you don't pass a prop into a component, it will default to `null`, which is a falsey value in JavaScript.
```

Comment:
This is incorrect, the missing value is `undefined` (which is what you get when you read a non-existent property on any object).